### PR TITLE
fix: app store spacing issues

### LINKED ISSUE #6336: [CAL-710] /App store : Spacing issues - Check all margins against design
* I have clearly spec'd all spacing in designs. Check all margin & spacing
* Also check the styles of the headers.

[View in Figma](https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=26%3A53096&t=AeM4S9etPe88eosE-1)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-710](https://linear.app/calcom/issue/CAL-710/app-store-spacing-issues-check-all-margins-against-design)</sub>

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

Fixes #6336

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by PeerRich** [comment]
@Jaibles can you test this preview, also on mobile and tablet?

**Comment by Jaibles** [comment]
Issues:

- [ ] [Missing the 12px margin below subheaders](https://share.cleanshot.com/bmtXxcvb)

**Comment by PeerRich** [comment]
oh yeah mobile looks odd, how should it be? @Jaibles 

I don't mind having it done in this PR

**Comment by Jaibles** [comment]
@PeerRich @G3root [mobile designs here](https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=1279%3A110032&t=WvpLGZDVXU50uT5E-1).

File: packages/features/shell/Shell.tsx [hunk 1]
```diff
@@ -148,6 +148,7 @@ type LayoutProps = {
   title?: string;
   heading?: ReactNode;
   subtitle?: ReactNode;
+  headerClassName?: string;
```
Comment (G3root) [inline packages/features/shell/Shell.tsx:151]: a workaround to hide title and subtitle in app pages on tablet

**Comment by G3root** [comment]
new changes

- [x] fixed Missing the 12px margin below subheaders
Screenshot 2023-01-11 at 22-48-10 App Store Cal com

- [x] slider button hover state
Screenshot 2023-01-11 22:52:17


- [x] category card
 Screenshot 2023-01-11 at 22-53-34 App Store Cal com

- [x] mobile and tablet view

**Comment by G3root** [comment]
@PeerRich marking this PR as draft because some other commits are added to this PR , similar to what happened with @sean-brydon . i'll make another PR with a clean branch.

**Pull request closed without merging**